client/src/component/MilestoneTable.js

```diff
@@ -11,13 +15,14 @@ const MilestoneTable = () => {
 				</thead>
 				<tbody>
 					<tr>
-						<td>5 out of 7</td>
-						<td>3</td>
+						<td>{githubData.total_count} out of 26</td>
+						<td>{codewarsData.ranks?.overall?.name || ""} </td>
+						{/* (data.ranks.overall.name) so if codewarsData . ranks is defined, check . overall, and if that’s defined, check
+						 . name, but if any of them are undefined, use the empty string on the right hand side of the ||” */}
```

Comment:
Its good to remove any comments that dont give extra information.

Reply:
Removed, I thought may other team members needed it.